### PR TITLE
Add JavaScript Question

### PR DESCRIPTION
**Pull request by kairules55**
Add JavaScript question

### CODE REVIEW

File: javascript/answers/Krishana-q7.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                <h4>
+                    <h4>What are JavaScript Promises?</h4>
+                </h4>
+            </div>
+            <br>
+            <br>
+            <div>
+                <strong>Answer:</strong> 
+                <p>
+                    A <b>Promise</b> is a proxy for a value not necessarily known when the promise is created. It allows you to associate handlers with an asynchronous action's eventual success value or failure reason. This lets asynchronous methods return values like synchronous methods: instead of immediately returning the final value, the asynchronous method returns a promise to supply the value at some point in the future.
```
Comment (shivamyth) [inline javascript/answers/Krishana-q7.html]: Elaborate with an example.

Comment (shivamyth) [inline javascript/answers/Krishana-q7.html:48]: Elaborate with an example

**Review by shivamyth — CHANGES_REQUESTED** [review]

File: javascript/answers/Krishana-q7.html [hunk 2]
```diff
@@ -51,6 +51,23 @@ <h4>What are JavaScript Promises?</h4>
                 <p>
                     A <b>Promise</b> is a proxy for a value not necessarily known when the promise is created. It allows you to associate handlers with an asynchronous action's eventual success value or failure reason. This lets asynchronous methods return values like synchronous methods: instead of immediately returning the final value, the asynchronous method returns a promise to supply the value at some point in the future.
                 </p>
+
```
Comment (kairules55) [inline javascript/answers/Krishana-q7.html:50]: given example for the question.

File: javascript/answers/Krishana-q7.html [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        gtag('config', 'UA-157837769-1');
+        </script>
+        
+        <title>
+            JavaScript | InterQues | Coding Ninjas
+        </title>
+
+        <link rel="shortcut icon" type="image/png" href="../../assets/favicon.png"/>
+
+        <meta name="description" content="What are escape Sequences?">
```
Comment (arpansac) [inline javascript/answers/Krishana-q7.html]: Please change to the question text in content=""

File: javascript/answers/Krishana-q7.html [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+            <img src="../../assets/CN_logo.png" height='72px' alt="Coding Ninjas Logo">
+        </div>
+
+        <div>
+            <p>
+                <small>
+
+                    <a href="../../index.html">Home</a>
+                    &gt; <a href="../index.html">JavaScript Questions</a>
+                    &gt; What are escape Sequences?
```
Comment (arpansac) [inline javascript/answers/Krishana-q7.html:32]: Same here

**Review by arpansac — CHANGES_REQUESTED** [review]
Please do the requested changes

File: javascript/answers/Krishana-q7.html (hunk 3 above)
Comment (dakshkhetan) [reply javascript/answers/Krishana-q7.html]: @kairules55 Any updates on this?

File: javascript/answers/Krishana-q7.html [hunk 5]
```diff
@@ -27,17 +27,13 @@
         <div>
             <p>
                 <small>
-
                     <a href="../../index.html">Home</a>
                     &gt; <a href="../index.html">JavaScript Questions</a>
                     &gt; What are escape Sequences?
```
Comment (dakshkhetan) [inline javascript/answers/Krishana-q7.html:32]: Edit line 32 and change this to question title.

**Review by dakshkhetan — CHANGES_REQUESTED** [review]

**Comment by dakshkhetan** [comment]
Closing this PR as no changes were made.

**Pull request closed without merging**